client_api.go

```diff
@@ -63,10 +70,16 @@ const (
 	endpointStatsRevoked                = "/stats/revoked"
 	endpointTrustChain                  = "/trustchain"
 	endpointPolicy                      = "/validationpolicy"
-	pathDNS                             = "/dns"
 	pathReassert                        = "/reassert"
 )
 
+var (
```

Comment:
can be included under const

Reply:
this has changed in the latest commit and it is under const